tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -111,7 +124,85 @@ class WskWebActionsTests
                 .config(sslconfig)
                 .auth().preemptive().basic(wskprops.authKey.split(":")(0), wskprops.authKey.split(":")(1))
                 .get(url)
-            authorizedResponse.statusCode shouldBe 200
-            authorizedResponse.body().asString() shouldBe namespace
+
+            if (testRoutePath == "/api/v1/experimental/web") {
+                authorizedResponse.statusCode shouldBe 200
+                authorizedResponse.body().asString() shouldBe namespace
+            } else {
+                authorizedResponse.statusCode shouldBe 401
```

Comment:
this is going to cause confusion - 401 when the request provides its auth token.
we can support it (as we do for v1), ignore it, or return different code.